lighthouse-core/computed/critical-request-chains.js

```diff
@@ -158,17 +206,40 @@ class CriticalRequestChains {
   }
 
   /**
-   * @param {{URL: LH.Artifacts['URL'], devtoolsLog: LH.DevtoolsLog}} data
+   * @param {{URL: LH.Artifacts['URL'], devtoolsLog: LH.DevtoolsLog, trace: LH.Trace}} data
    * @param {LH.Audit.Context} context
    * @return {Promise<LH.Artifacts.CriticalRequestNode>}
    */
   static async compute_(data, context) {
-    const [networkRecords, mainResource] = await Promise.all([
+    const [networkRecords, mainResource, graph] = await Promise.all([
       NetworkRecords.request(data.devtoolsLog, context),
       MainResource.request(data, context),
+      PageDependencyGraph.request(data, context),
     ]);
 
-    return CriticalRequestChains.extractChain(networkRecords, mainResource);
+
+    // debugging.
+    const a = await CriticalRequestChains.extractChain(networkRecords, mainResource);
```

Comment:
Will remove. Left in for reviewers to easily compare like I did when developing.